api/specs/cee-v3.0.yaml

```
@@ -0,0 +1,494 @@
+                '$ref': '#/components/schemas/cee_application_identity_import'
+      responses:
+        '200':
+          description: Ok
+          content:
+            'application/json':
+              schema:
+                '$ref': '#/components/schemas/cee_application_identity_import_response'
+  /policies/cee/{uuid}:
+    get:
+      tags:
+      - cee
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)